cstore_fdw.c

```diff
@@ -55,15 +55,41 @@
 #include "utils/tqual.h"
 
 
+#if PG_VERSION_NUM < 100000
+/*
+ * QueryEnvironment definition only exist in PG_VERSION >= 10
+ * Here is a dummy definition to enable building in previous versions.*/
+typedef void QueryEnvironment;
```

Comment:
You no longer need this now that you've moved every instance of QueryEnvironment into an `#if PG_VERSION_NUM`